proto/internalapi/sensor/compliance_nodescan.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option go_package = "sensor";
+
+import "google/protobuf/timestamp.proto";
+import "scanner/api/v1/note.proto";
+import "scanner/api/v1/component.proto";
+
+
+import "gogoproto/gogo.proto";
+
+package sensor;
+
+// Names and data types are designed to follow proto/storage/node.proto as close as possible
+// Next tag: 7
+message FullNodeScan {
+    string                         id         = 1 [(gogoproto.moretags) = 'search:"Node ID,store" sql:"pk"'];
+    string                         name       = 2 [(gogoproto.moretags) = 'search:"Node,store"'];
```

Comment:
Same for node name
```suggestion
    string                         node_name       = 2 [(gogoproto.moretags) = 'search:"Node,store"'];
```

Reply:
My logic is that "NodeScan" combines two things: 1) information about the node, 2) information about the scan. While information about the scan is more or less self-evident when you look at the message, information about the node isn't. Just `name` and `id` could mean anything: "name" and "id" of the scan or "name" and "id" of the node, or "name" of the node but "id" of the scan. Therefore I suggest to make things more explicit and clarify directly in the name.

The example of `Node` and `NodeScan` doesn't match here. `Node` rightfully has `id` and `name` because there hardly can be a confusion what these mean. `NodeScan` at the same time doesn't have `id` or `name` attributes, unlike the new `NodeScanV2`.